### ISSUE TITLE
Multitouch gestures on macOS not supported on client

### ISSUE THREAD

**Issue by cosmos503** [debauchee/barrier#591]
### Operating Systems ###

Server: macOS 10.15.3

Client: macOS 10.15.3

### Barrier Version ###

2.3.2-Release-210c2b70

### Steps to reproduce bug ###

Barrier works fine on client for common usage, but multitouch trackpad gestures for macOS are not working on the client, but work fine on the server. Specifically the four-finger left-right to move between Spaces and the four-finger up gesture for Expose.

**Comment by squizzeak** [comment]
This is also true for magic mouse gestures.

**Comment by kreisys** [comment]
This is also true for synergy and they've been procrastinating that for almost a decade so I'm guessing there's an actual blocker here.

**Comment by findmory** [comment]
I use Better Touch Tool to receive the trackpad gestures on my host machine and send the commands to the client.  Pretty decent workaround.  Feel free to follow instructions using [this gist](https://gist.github.com/findmory/82593b407ef436fe8a39f1bbb6802690) I created.

**Comment by kreisys** [comment]
Nope mr bot this is still pretty much desired

**Comment by mono424** [comment]
yes should be considered, its pretty painful to switch from gestures to keyboard shortcuts all the time :)

**Comment by OKNoah** [comment]
Does this happen with Synergy or whatever? It looks like the solutions involve paying for something. Pity.

**Comment by OKNoah** [comment]
@mono424 @kreisys @findmory Just learned this software is likely abandoned and likely don't see new features of fixes that aren't community submitted.

**Comment by p12tic** [comment]
> Just learned this software is likely abandoned and likely don't see new features of fixes that aren't community submitted.

This is true for about any kind of open-source project which is not supported by commercial contributors.

**Comment by zeves095** [comment]
so... no gestures?

**Comment by delCatta** [comment]
:(

**Comment by s-daveb** [comment]
Interestingly enough, with the latest released versions, on macOS:

 I can scroll to the side with two-finger touch gestures when I use a Macbook Air _trackpad_, but if I use a magic mouse on my Macbook Pro to control my Macbook Air client, there is no side scrolling.
